book/04-git-server/sections/protocols.asc

```diff
@@ -1,100 +1,100 @@
+Tatalakayin natin kung papaano mag-export ng isang kopya ng hubad na repositoryo para sa itong layunin sa <<_git_on_the_server#_getting_git_on_a_server>>.
 
-This is also a nice option for quickly grabbing work from someone else's working repository.
-If you and a co-worker are working on the same project and they want you to check something out, running a command like `git pull /home/john/project` is often easier than them pushing to a remote server and you subsequently fetching from it.
+Ito rin ay isang magandang opsyon para sa mabilis na pagkuha ng trabaho mula sa gumagana na repositoryo ng iba.
+Kapag ikaw at ang isang co-worker ay nagtatrabaho sa isang proyekto at gusto nila na may kunin ka, ang pagpapagana ng isang utos tulad ng `git pull /home/john/project` ay kadalasan mas madali kesa mag-push sila sa isang remote na server at pagkatapos ay kukunin mo.
 
-===== The Cons
 
-The cons of this method are that shared access is generally more difficult to set up and reach from multiple locations than basic network access.
-If you want to push from your laptop when you're at home, you have to mount the remote disk, which can be difficult and slow compared to network-based access.
+===== Ang mga Kahinaan
+
+Ang mga kahinaan ng paraan na ito ay ang mga ibinahagi na access ay sa pangkalahatan mas mahirap i-set up at maabot mula sa maramihang mga lokasyon kaysa sa pangunahing network access.
+Kapag gusto mong mag-push mula sa iyong laptop kapag ikaw ay nasa bahay, kailangan mong i-mount ang remote dist, na maaaring maging mahirap at mahina kumpara sa batay sa network na access.
 
-It's important to mention that this isn't necessarily the fastest option if you're using a shared mount of some kind.
-A local repository is fast only if you have fast access to the data.
-A repository on NFS is often slower than the repository over SSH on the same server, allowing Git to run off local disks on each system.
+Mahalagang banggitin na hindi ito ang pinakamabilis na opsyon kung ikaw ay gumagamit ng isang uri ng ibinahagi na mount.
+Ang isang lokal na repositoryo ay mabilis lamang kapag ikaw ay may mabilis na access sa data.
+Ang repositoryo na nasa NFS ay madalas mas mahina kumpara sa repositoryo na nasa SSH sa parehong server, nagpapahintulot na ipagana ang Git gamit ang lokal na mga disk sa bawat sistema.
 
-Finally, this protocol does not protect the repository against accidental damage.
-Every user has full shell access to the ``remote'' directory, and there is nothing preventing them from changing or removing internal Git files and corrupting the repository.
+Sa wakas, hindi pinoprotektahan ng protokol na ito ang repositoryo mula sa mga hindi sinasadya na mga aksidente.
+Ang bawa't gumagamit ay may buong access ng shell sa ``remote'' na direktoryo, at walang pumipigil sa kanila sa pag bago o pag tanggal ng mga Git files sa loob at pag-corrupt sa repositoryo.
```

Comment:
* ``bawa't`` to ``bawat``
* ``pag bago`` to ``pagbago``
* ``pag tanggal`` to ``pagtanggal``